e2e-tests/playwright.config.ts

```diff
@@ -75,7 +76,10 @@ export default defineConfig({
     },
     {
       name: "showcase-auth-providers",
-      testMatch: ["**/playwright/e2e/authProviders/*.spec.ts"],
+      testMatch: [
+        "**/playwright/e2e/authProviders/*.spec.ts",
+        "**/playwright/e2e/plugins/github-insights/*.spec.ts",
```

Comment:
The auth-providers job is not the best place to run any GitHub plugin-related test; this job is going to be refactored soon and will not include any plugin test. Also, just adding the files in testMatch list is not enough to run them in this job, as they do not run against a single RHDH instance, but multiple ones are created and deleted dynamically during the tests execution. 
I would recommend to setup the Github integration and the Github auth provider in the e2e main job configuration files and run the plugins tests along with the other plugins tests.

Reply:
Hi @albarbaro, any update on this?